### PR TITLE
Fix to Missing details how to deploy mongodb to heroku 

### LINKED ISSUE #8444: No details how to deploy mongodb to heroku
Challenge [URL Shortener Microservice](https://www.freecodecamp.com/challenges/url-shortener-microservice) has an issue.

There are no Details for deploying mongodb in heroku.. Its my first time i am trying to deploy a db into heroku and i am finding it hard.

At last i found out a way to do this is adding mLab MongoDB add-on but it is asking for a credit card for billing , Not all people will have a credit card, Is there any better alternative for this? or anyother better way to do this Challenge other than using a DB.

or if in case i am assuming this in wrong way please tell me and close this issue.
my code is at `https://github.com/ChaituVR/URL-Shortner.git`

image

Thank you

### PR DESCRIPTION
**Pull request by ChaituVR**
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [x] Closes currently open issue (`Closes #XXXX`): Closes #8444
#### Description

Added a line:

ProTip:Checkout this article on the wiki for tips on integrating MongoDB on heroku

### CODE REVIEW

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 1]
```diff
@@ -258,6 +258,7 @@
         "<strong>User Story:</strong>  I can pass a URL as a parameter and I will receive a shortened URL in the JSON response.",
         "<strong>User Story:</strong> If I pass an invalid URL that doesn't follow the valid http://www.example.com format, the JSON response will contain an error instead.",
         "<strong>User Story:</strong> When I visit that shortened URL, it will redirect me to my original link.",
+        "<strong>Pro Tip:</strong> Checkout this <a href='https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Using-MongoDB-And-Deploying-To-Heroku' targer='_blank'>article on the wiki</a> for tips on integrating MongoDB on Heroku.",
```
Comment (theflametrooper) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: ~~targer='_blank'~~ target='_blank'

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 2]
```diff
@@ -258,6 +258,7 @@
         "<strong>User Story:</strong>  I can pass a URL as a parameter and I will receive a shortened URL in the JSON response.",
         "<strong>User Story:</strong> If I pass an invalid URL that doesn't follow the valid http://www.example.com format, the JSON response will contain an error instead.",
         "<strong>User Story:</strong> When I visit that shortened URL, it will redirect me to my original link.",
+        "<strong>Pro Tip:</strong> Checkout this <a href='https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Using-MongoDB-And-Deploying-To-Heroku' target='_blank'>article on the wiki</a> for tips on integrating MongoDB on Heroku.",
```
Comment (theflametrooper) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: ~~article on the wiki~~ wiki article

**Comment by ChaituVR** [comment]
Can some when help me squashing that `Update CONTRIBUTING.md`

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **PERFECT.** :sparkles: :sparkles: 

Thanks for you contribution @ChaituVR, looking forward to more!

**Comment by ChaituVR** [comment]
Thanks to you @raisedadead for suggesting this  :)

**Pull request merged**